properties.schema

```diff
@@ -77,6 +77,15 @@
       "help": "This is the instruction text",
       "translatable": true
     },
+    "ariaQuestion": {
+      "type": "string",
+      "required": false,
+      "default": "",
+      "inputType": "Text",
+      "validators": [],
+      "help": "This will be read out by screen readers instead of reading the 'Display title', 'Body' & 'Instruction' fields when focusing on the options. To be clear and concise, ensure the text encompasses only the question associated.",
```

Comment:
Should this help text me more similar to the text in the read me, so it might say
`This will be read out by screen readers instead of reading the title, body & instruction fields when focusing on the group or radiogroup. To be clear and concise, ensure the text encompasses only the question associated.` instead, replacing "when focusing on the options" with "when focusing on the group or radiogroup".